Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelListItem.swift

```diff
@@ -127,6 +130,16 @@ public struct ChatChannelListItem: View {
     }
 }
 
+/// Options for setting up the channel avatar view.
+public struct ChannelAvatarViewOptions {
+    /// Whether the online indicator should be shown.
+    public var showOnlineIndicator: Bool
+    /// Size of the avatar.
+    public var size: CGSize = .defaultAvatarSize
+    /// Optional avatar image. If not provided, it will be loaded by the channel header loader.
+    public var avatar: UIImage?
+}
```

Comment:
I think we need to make the init public as well right? 🤔

Reply:
as discussed, we're keeping it internal for now.